packages/webchat/src/core/socket.tsx

```diff
@@ -10,6 +10,8 @@ export default class BpSocket {
   constructor(config: Config) {
     this.chatId = config.chatId
     this.socket = new MessagingSocket({ url: config.messagingUrl, clientId: config.clientId })
+
+    window.websocket = this.socket
```

Comment:
Not 100% sure about this, but I haven't found any other clean way of accessing this class inside the tests.